tests/monotouch-test/Compression/CompressionStreamTest.cs

```diff
@@ -70,6 +74,7 @@ public void CheckCompressDecompress (CompressionAlgorithm algorithm)
 			DeflateStream decompressing = new DeflateStream (backing, CompressionMode.Decompress, algorithm);
 			MemoryStream output = new MemoryStream ();
 			CopyStream (decompressing, output);
+			Assert.AreNotEqual (0, output.Length);
 			Assert.IsTrue (compare_buffers (data, output.GetBuffer(), (int) output.Length));
```

Comment:
whenever there's more than one assert there should be a unique message with them

Reply:
Sure, remember that this is comming from mono. The only reason it was not done is because I wanted to keep the diff to the min. Nevertheless, I'll update the tests.